### PR TITLE
Fix unclosed `console.group` and overhaul logging in `loader.ts`

### LINKED ISSUE #6899: IDE logging not working when opening cloud project
### Discord username

_No response_

### What type of issue is this?

Permanent – Occurring repeatably

### Is this issue blocking you from using Enso?

- [ ] Yes, I can't use Enso because of this issue.

### Is this a regression?

- [ ] Yes, previous version of Enso did not have this issue.

### What issue are you facing?

When opening a cloud project in the IDE (started with `./run ide watch --backend-source release --backend-release latest --skip-wasm-opt --ide-option -debug.dev-tools=true --ide-option -debug`), there is no debug output in the dev console (e.g., logs from calls to `console_log!`). 
It works fine when opening a local project. 

### Expected behaviour

Log output should appear. 

### How we can reproduce it?

* Run `./run ide watch --backend-source release --backend-release latest --skip-wasm-opt --ide-option -debug.dev-tools=true --ide-option -debug `
* Switch to cloud projects in the dashboard
* Open a cloud project
* Watch the dev console

### Screenshots or screencasts

Local project output (with IDE log output)
image

Cloud project output (missing IDE log output)
image

### Logs

_No response_

### Enso Version

develop

### Browser or standalone distribution

Standalone distribution (cloud project)

### Browser Version or standalone distribution

standalone

### Operating System

Linux

### Operating System Version

_No response_

### Hardware you are using

_No response_

### PR DESCRIPTION
**Pull request by somebody1234**
### Pull Request Description
- Fixes #6899

### Important Notes
The linked issue is actually a combination of two issues - but the other (the cloud IDE using a different version of `pkg.wasm`) is unsolvable as that is intended behavior

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] ~~The documentation has been updated, if necessary.~~
- [x] ~~Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.~~
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [x] ~~Unit tests have been written where possible.~~
  - [x] ~~If GUI codebase was changed, the GUI was tested when built using `./run ide build`.~~

### CODE REVIEW

**Review by wdanilo — APPROVED** [review]
Looks good, but please do in-depth QA that everything loads correctly after these changes :)

**Comment by indiv0** [comment]
**TL;DR: QA passed successfully**

# Scenario 0

```sh
./run gui watch \
    --skip-wasm-opt \
    --skip-version-check
```

- ✅ GUI at http://localhost:8080 opens
- ✅ Can login
- ✅ Dashboard shows up

# Scenario 1

```sh
ENSO_BUILD_TARGET=dir \
    ./run ide watch \
    --skip-wasm-opt \
    --wasm-profile=dev \
    --backend-source=release \
    --backend-release=nightly \
    --skip-version-check \
    --ide-option=-feature-preview.new-dashboard \
    --ide-option=-debug.dev-tools \
    --ide-option=-debug \
    --ide-option=--inspect=5858 \
    --ide-option=-authentication \
    --ide-option=-startup.entry=text_area
```

- ✅ Debug scene is visible
- ✅ Logs appear with `console.log("Hello")`

<details>
<summary>
Screenshot
</summary>
<pre>
image
</details>

# Scenario 2

```sh
ENSO_BUILD_TARGET=dir \
    ./run ide watch \
    --skip-wasm-opt \
    --wasm-profile=dev \
    --backend-source=release \
    --backend-release=nightly \
    --skip-version-check \
    --profiling-level=objective \
    --wasm-log-level=info \
    --wasm-uncollapsed-log-level=info \
    --ide-option=-feature-preview.new-dashboard \
    --ide-option=-debug.dev-tools \
    --ide-option=-debug \
    --ide-option=--inspect=5858 \
    --ide-option=-authentication
```

- Enabled logs at https://github.com/enso-org/enso/blob/cfb2f2916eb1d81f521c5af238e42c2b227f65d3/app/gui/view/graph-editor/src/component/visualization/container.rs#L128 by replacing:
 ```rust
 ensogl::define_endpoints! {
 ```
 with:
 ```rust
 ensogl::define_endpoints! { [TRACE_ALL]
```

- ✅ Nodes in Cloud project are visible
- ✅ `[INFO]` logs appear in console
- ✅ Logs appear with `console.log("Hello")`
- ✅ FRP logs appear in the output

<details>
<summary>
Screenshot
</summary>
image
</details>

# Scenario 3

```sh
ENSO_BUILD_TARGET=dir \
    ./run ide watch \
    --skip-wasm-opt \
    --wasm-profile=dev \
    --backend-source=release \
    --backend-release=nightly \
    --skip-version-check \
    --profiling-level=objective \
    --wasm-log-level=info \
    --wasm-uncollapsed-log-level=info \
    --ide-option=-debug.dev-tools \
    --ide-option=-debug \
    --ide-option=--inspect=5858 \
    --ide-option=-startup.entry=text_area
```

- ✅ Debug scene is visible
- ✅ Logs appear with `console.log("Hello")`

<details>
<summary>
Screenshot
</summary>
image
</details>

# Scenario 4

```sh
ENSO_BUILD_TARGET=dir \
    ./run ide watch \
    --skip-wasm-opt \
    --wasm-profile=dev \
    --backend-source=release \
    --backend-release=nightly \
    --skip-version-check \
    --profiling-level=objective \
    --wasm-log-level=info \
    --wasm-uncollapsed-log-level=info \
    --ide-option=-debug.dev-tools \
    --ide-option=-debug \
    --ide-option=--inspect=5858
```

- Enabled logs at https://github.com/enso-org/enso/blob/cfb2f2916eb1d81f521c5af238e42c2b227f65d3/app/gui/view/graph-editor/src/component/visualization/container.rs#L128 by replacing:
 ```rust
 ensogl::define_endpoints! {
 ```
 with:
 ```rust
 ensogl::define_endpoints! { [TRACE_ALL]
```

- ✅ Nodes in Cloud project are visible
- ✅ `[INFO]` logs appear in console
- ✅ Logs appear with `console.log("Hello")`
- ✅ FRP logs appear in the output

<details>
<summary>
Screenshot
</summary>
image
</details>

# Scenario 5

```sh
ENSO_BUILD_TARGET=dir \
    ./run ide build \
    --skip-wasm-opt \
    --wasm-profile=dev \
    --backend-source=release \
    --backend-release=nightly \
    --skip-version-check \
    --profiling-level=objective \
    --wasm-log-level=info \
    --wasm-uncollapsed-log-level=info
./dist/ide/mac-arm64/Enso.app/Contents/MacOS/Enso \
    -feature-preview.new-dashboard \
    -debug.dev-tools \
    -debug \
    --inspect=5858 \
    -authentication
```

<details>
<summary>
Screenshot
</summary>
image

</details>

# Scenario 6

```sh
cd app/ide-desktop
npm run build-dashboard
```

- ✅ Command ran successfully

<details>
<summary>
Screenshot
</summary>
image
</details>

# Scenario 7

```sh
cd app/ide-desktop
npm run watch-dashboard
```

- ✅ Command ran successfully

<details>
<summary>
Screenshot
</summary>
image
</details>
